tests/transfer.test.js

```diff
@@ -0,0 +1,31 @@
+// account.test.js
+import { describe, it, expect } from 'vitest';
+import { Account } from '../src/model/Account.js';
+
+describe('deposit', () => {
+    it('Should transfer a valid amount', () => {

```

Comment:
some test cases are missing, like when the transfer amount exceeds the current balance or when the account is the same